ddtrace/internal/processor/trace.py

```diff
@@ -68,6 +68,33 @@ def process_trace(self, trace):
         return None
 
 
+@attr.s
+class TraceTopLevelSpanProcessor(TraceProcessor):
+    """Processor marks spans as top level"""
+
+    def process_trace(self, trace):
+        # type: (List[Span]) -> Optional[List[Span]]
+        """Mark a span in a trace as top level if:
+         1. Span has a different service name than it's parent
+         2. Span is a local root
```

Comment:
```suggestion
         1. Span is a local root
         2. Span has a different service name than it's parent
```

Just a small nit to swap the order to match the if statement below, e.g. 1. is more important than 2.